crypto-ffi/bindings/js/wdio.conf.ts

```diff
@@ -1,14 +1,18 @@
+import { tmpdir } from "os";
 import fs from "fs/promises";
 import path from "path";
 import { fileURLToPath } from "url";
 import logger from "@wdio/logger";
 
-const STATIC_SEVER_URL = "http://localhost:3000/";
-
-const staticPath = path.resolve(process.cwd(), `../platforms/web`);
-const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const STATIC_SERVER_URL = "http://localhost:3000/";
 const log = logger("wdio.conf.ts");
 
+let staticPath;
+if (staticPath === undefined) {
+    staticPath = await fs.mkdtemp(path.join(tmpdir(), 'core-crypto-wdio-test-'));
```

Comment:
Intentional trailing hyphen?

Reply:
> Intentional trailing hyphen?

Yes, those typically end up being of the form `core-crypto-wdio-test-XYZABC`, so having the hyphen makes it slightly easier to read, if there ever is a need to interact with those files.